run-integration-tests.bat

```diff
@@ -4,4 +4,10 @@
 go build -tags=forceposix -o build/sourced-ce_windows_amd64/sourced.exe ./cmd/sourced
 
 :: run tests
+set TMPDIR_INTEGRATION_TEST=C:\tmp
+:: see https://stackoverflow.com/questions/22948189/batch-getting-the-directory-is-not-empty-on-rmdir-command
+del /f /s /q %TMPDIR_INTEGRATION_TEST% 1>nul
+rmdir /s /q %TMPDIR_INTEGRATION_TEST%
+mkdir %TMPDIR_INTEGRATION_TEST%
+set TMP=%TMPDIR_INTEGRATION_TEST%

```

Comment:
where do we use `TMP` env var?

Reply:
When calling `ioutil.TempDir`. For unix the default location is given by the `TMPDIR` env var, and we set it in the `test-integration-clean` target of the Makefile. For windows, the first option is the `TMP` env var [link](https://golang.org/pkg/os/#TempDir).